app/controllers/home_controller.rb

```diff
@@ -1,10 +1,13 @@
 class HomeController < ApplicationController
   before_action :fetch_va_facilities, only: [:index, :diffusion_map]
+  include CategoryUsageUtils
```

Comment:
Do we still need this?

Reply:
Leaving as a placeholder for "most popular categories"  query that we will need in the new home page feature branch.